tutorials/clustering.md

```diff
@@ -8,9 +8,12 @@ TimescaleDB can be run in a multi-node setup, with one primary access node distr
 writes and queries to multiple data nodes. For more information on the actual
 underlying architecture, you can view our [architecture][architecture] section.
 
-In this tutorial, we will show how you can create a distributed hypertable within
-a distributed database. We will set up the distributed database with one primary access node
-and two data nodes across three distinct TimescaleDB instances.
+In this tutorial, we will walk through setting up a three node distributed
+database and creating a distributed hypertable. As part of this we will show
+you how to enable `scram-sha-256` authentication and add new roles to your
+distributed database.
```

Comment:
Suggestion: "This tutorial will walk you through setting up a three-node distributed database and create a distributed hypertable. You will learn how to enable password authentication and add new roles to your distributed database."